react/Checkbox/index.jsx

```diff
@@ -19,6 +21,7 @@ const Checkbox = props => {
     <label
       className={cx(
         styles['c-input-checkbox'],
+        styles['c-input-checkbox--svg'],
```

Comment:
What are the cases where we do not have the --svg modifier ?

Reply:
You'll get what we have today : an icon with `embedurl`